[2019-11-17 18:12:23,384] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: dend_dag.Load_time_dim_table 2018-11-03T04:00:00+00:00 [queued]>
[2019-11-17 18:12:23,427] {taskinstance.py:620} INFO - Dependencies all met for <TaskInstance: dend_dag.Load_time_dim_table 2018-11-03T04:00:00+00:00 [queued]>
[2019-11-17 18:12:23,428] {taskinstance.py:838} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 18:12:23,429] {taskinstance.py:839} INFO - Starting attempt 1 of 4
[2019-11-17 18:12:23,429] {taskinstance.py:840} INFO - 
--------------------------------------------------------------------------------
[2019-11-17 18:12:23,463] {taskinstance.py:859} INFO - Executing <Task(LoadDimensionOperator): Load_time_dim_table> on 2018-11-03T04:00:00+00:00
[2019-11-17 18:12:23,464] {base_task_runner.py:133} INFO - Running: ['airflow', 'run', 'dend_dag', 'Load_time_dim_table', '2018-11-03T04:00:00+00:00', '--job_id', '275', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/dend_dag.py', '--cfg_path', '/var/folders/8q/nyxhvd6d55nfqjdbrybk6jpw0000gn/T/tmpa80dhk6w']
[2019-11-17 18:12:30,270] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table /Users/likejun/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
[2019-11-17 18:12:30,272] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   """)
[2019-11-17 18:12:30,902] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table [2019-11-17 18:12:30,900] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-11-17 18:12:33,782] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table [2019-11-17 18:12:33,780] {dagbag.py:90} INFO - Filling up the DagBag from /Users/likejun/airflow/dags/dend_dag.py
[2019-11-17 18:12:33,876] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table [2019-11-17 18:12:33,875] {cli.py:516} INFO - Running <TaskInstance: dend_dag.Load_time_dim_table 2018-11-03T04:00:00+00:00 [running]> on host likejundeMacBook-Pro.local
[2019-11-17 18:12:33,928] {load_dimensions.py:23} INFO - LoadDimensionOperator for time_dim
[2019-11-17 18:12:33,970] {logging_mixin.py:95} INFO - [[34m2019-11-17 18:12:33,968[0m] {[34mbase_hook.py:[0m84} INFO[0m - Using connection to: [1mid: redshift. Host: redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com, Port: 5439, Schema: dev, Login: awsuser, Password: XXXXXXXX, extra: {}[0m[0m
[2019-11-17 18:12:33,976] {taskinstance.py:1051} ERROR - could not translate host name "redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com" to address: nodename nor servname provided, or not known
Traceback (most recent call last):
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 38, in execute
    pg_hook.run(f"DELETE FROM {self.table}")
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
    with closing(self.get_conn()) as conn:
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/postgres_hook.py", line 75, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/Users/likejun/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 130, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com" to address: nodename nor servname provided, or not known

[2019-11-17 18:12:33,984] {taskinstance.py:1074} INFO - Marking task as UP_FOR_RETRY
[2019-11-17 18:12:34,043] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table Traceback (most recent call last):
[2019-11-17 18:12:34,044] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/bin/airflow", line 32, in <module>
[2019-11-17 18:12:34,045] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     args.func(args)
[2019-11-17 18:12:34,045] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-11-17 18:12:34,045] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     return f(*args, **kwargs)
[2019-11-17 18:12:34,046] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
[2019-11-17 18:12:34,046] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     _run(args, dag, ti)
[2019-11-17 18:12:34,046] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
[2019-11-17 18:12:34,047] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     pool=args.pool,
[2019-11-17 18:12:34,047] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
[2019-11-17 18:12:34,047] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     return func(*args, **kwargs)
[2019-11-17 18:12:34,048] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 926, in _run_raw_task
[2019-11-17 18:12:34,048] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     result = task_copy.execute(context=context)
[2019-11-17 18:12:34,048] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/airflow/plugins/operators/load_dimensions.py", line 38, in execute
[2019-11-17 18:12:34,049] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     pg_hook.run(f"DELETE FROM {self.table}")
[2019-11-17 18:12:34,049] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
[2019-11-17 18:12:34,049] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     with closing(self.get_conn()) as conn:
[2019-11-17 18:12:34,049] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/airflow/hooks/postgres_hook.py", line 75, in get_conn
[2019-11-17 18:12:34,050] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     self.conn = psycopg2.connect(**conn_args)
[2019-11-17 18:12:34,050] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table   File "/Users/likejun/anaconda3/lib/python3.7/site-packages/psycopg2/__init__.py", line 130, in connect
[2019-11-17 18:12:34,050] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table     conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
[2019-11-17 18:12:34,051] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table psycopg2.OperationalError: could not translate host name "redshift-cluster.chjcdyihped1.us-west-2.redshift.amazonaws.com" to address: nodename nor servname provided, or not known
[2019-11-17 18:12:34,051] {base_task_runner.py:115} INFO - Job 275: Subtask Load_time_dim_table 
[2019-11-17 18:12:38,699] {logging_mixin.py:95} INFO - [[34m2019-11-17 18:12:38,696[0m] {[34mlocal_task_job.py:[0m105} INFO[0m - Task exited with return code 1[0m
